internal/utils/config.go

```diff
@@ -460,6 +464,24 @@ type (
 	// }
 )
 
+func (h *hookConfig) HandleHook(hookType string) error {
+	// If not enabled do nothing
+	if !h.Enabled {
+		return nil
+	}
+	if h.URI == "" {
+		return fmt.Errorf("missing required field in config: auth.hook.%s.uri", hookType)
```

Comment:
```suggestion
		return errors.Errorf("missing required field in config: auth.hook.%s.uri", hookType)
```

use `go-errors` package for auto stack trace.

Reply:
Thanks! TIL